### DISCUSSION TITLE
Incorrect Flickr URL being generated

### DISCUSSION THREAD

**Discussion by eDad2003** [Nachtzuster/BirdNET-Pi#132] [General]
First of all, I'm so happy to have found this repo!  It's really great that BirdNet-Pi will live on!

I was wondering if this is problem is experienced by anyone else?

I added the Flickr API yesterday, so I now have inline images associated with detections.  When the image is clicked, you get a popup window with an expanded image including 3 URLs - Image link, Author link, and License URL. The 'Image link' is always reported by Flickr as 'not found'.

By playing around a bit, I determined that the URL is generated incorrectly - the image is not at AuthorURL+ID.  See below for samples.

I traced it back to PHP in overview.php and todays_detections.php.  Here's the code I changed to correct it:
`$modaltext = $flickr_cache["author_url"] . "/" . $flickr_cache["id"];`
`//next line added by me to correct Flickr 'Image Link' URL`
`$modaltext = str_replace("people", "photos", $modaltext);`

It works... kinda.  There must be some caching or something else going on. I manually edit those PHP files (via file manager), save it, and then do a system reboot, but sometimes the incorrect version is still present. I'm kinda confused, but I'm new to this.

**Working URL - After code change**
![correct flickr](https://github.com/Nachtzuster/BirdNET-Pi/assets/49699285/434dab99-e49e-4903-9167-45cbba6579e4)

**Incorrect URL - Before code change**
![incorrect flickr](https://github.com/Nachtzuster/BirdNET-Pi/assets/49699285/a080dec7-750a-4c54-b0df-374e78a792c9)

**Comment by Nachtzuster** [comment]
Thanks, that seems to be a bug.
see: https://github.com/Nachtzuster/BirdNET-Pi/issues/135
